src/verify_face_auth.js

```diff
@@ -0,0 +1,35 @@
+import Syncano from 'syncano-server';
+
+export default (ctx) => {
+  const {response, users } = Syncano(ctx);
+
+  const {username, token} = ctx.args;
```

Comment:
Do we need the username here and as a param in the yml (same goes for the remove_face_auth)?

 `users.where('user_key', token)`

Reply:
True, not needed

Reply:
Seems searching by `user_key` does not work even after adding it to socket YAML and  setting the `filter_index` to `true`